packages/runner-ct/src/iframe/iframes.jsx

```diff
@@ -45,7 +45,8 @@ export default class Iframes extends Component {
           style={{
             height: viewportHeight,
             width: viewportWidth,
-            transform: `scale(${scale})`,
+            transform: `scale(${screenshotting ? 1 : scale})`,
+            transformOrigin: `${screenshotting ? 'top left' : ''}`,
```

Comment:
```transform: !screenshotting ? `scale(${scale}` : undefined```

And I don't think you'll need `transformOrigin`

Reply:
You were right - I didn't need `transformOrigin`. Nice catch.

Not sold on the `transform` refactor. Isn't an explicit `scale(1)` much more clear? Also, I think that `if X then Y else Z` is more clear than `if not X then Z else Y` syntax - ie, putting a negative at the start of the ternary feels a little weird, at least to me.

Reply:
Why emit HTML when you could not?

I suggested the ternary be written like that because I don't like having a falsy case be in the middle (the true case). Just a weird style preference of mine.